pkg/logql/downstream_test.go

```diff
@@ -181,6 +181,70 @@ func TestMappingEquivalenceSketches(t *testing.T) {
 	}
 }
 
+func TestMappingEquivalenceSketches_Instant(t *testing.T) {
```

Comment:
What makes this an instant query? What do you think about song another loop to the original equivalence tests?

Reply:
The query params right after this line, in the case of an instant query we set `start` and `end` to the same value.